.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '00 19 * * *'
```

Comment:
What does this mean?

Reply:
It's arbitrarily set to 7PM, but should probably be `00 12 * * *` (noon). (NTS when this is picked back up)